hive-metastore/src/test/java/org/apache/iceberg/hive/TestHiveCatalog.java

```diff
@@ -257,21 +259,42 @@ public void testReplaceTxnBuilder() throws Exception {
   public void testCreateTableWithOwner() throws Exception {
     Schema schema = getTestSchema();
     PartitionSpec spec = PartitionSpec.builderFor(schema).bucket("data", 16).build();
-    TableIdentifier tableIdent = TableIdentifier.of(DB_NAME, "tbl");
-    String location = temp.newFolder("tbl").toString();
     String owner = "some_owner";
-    ImmutableMap<String, String> properties =
-        ImmutableMap.of(TableProperties.HMS_TABLE_OWNER, owner);
-
+    Map<String, String> prop = ImmutableMap.of(TableProperties.HMS_TABLE_OWNER, owner);
+
+    createTableAndVerifyOwnership(DB_NAME, "tbl", schema, spec, prop, owner);
+
+    prop = ImmutableMap.of();
+
+    // a table without an owner explicitly specified should have a default owner
+    createTableAndVerifyOwnership(
+        DB_NAME,
+        "no_explicit_owner_specified_tbl",
+        schema,
+        spec,
+        prop,
```

Comment:
NIt, can we just pass ImmutableMap.of() here?

Reply:
This change is no longer needed